### ISSUE TITLE
[Cervantes]: unreliable wireless connection

### ISSUE THREAD

**Issue by pazos** [koreader/koreader#4313]
* KOReader version: lastest
* Device: BQ Cervantes 4. 

#### Issue
Sometimes when enabling wireless connection I got the error message described in https://github.com/koreader/koreader/blob/master/frontend/ui/network/wpa_supplicant.lua#L8. It seems weird because I'm able to connect *always* if I run ./enable-wifi.sh and then use wpa_cli to monitor wpa_supplicant while enabling wireless in KOReader.

**Comment by pazos** [comment]
I found what was happening with the obsolete version of wpa_supplicant bundled with debian wheezy (1.0.3). As you can see in its [manpage](https://manpages.debian.org/wheezy/wpasupplicant/wpa_supplicant.8.en.html) it does not support the override ctrl_interface `-O` and seems to ignore the ctrl interface inside the config file. The only option for that is use the `-C` command line option, which is incompatible with using a config file (`c`).

In short, skip generating a config file (since the file is not updated by lj-wpasupplicant and stock uses connman instead) and provide a ctrl interface with `-C` instead of `-O`.
